### ISSUE TITLE
Not available in the actions marketplace

### ISSUE THREAD

**Issue by EHYPI** [MicrosoftEdge/action-issue-to-workitem#1]
I could not find this when searching the github actions marketplace. Will this be released to the public?

**Comment by captainbrosset** [comment]
No, we have no plans to make this available on the marketplace. This is used for our own needs.
